p4-16/psa/PSA.mdk

```diff
@@ -405,27 +417,36 @@ numerical translation between controller port id values and data plane
 port id values, for each of the channels of communication between the
 controller and data plane described above.
 
-Occurrences of values of type `PortId_t` or `PortIdInHeader_t` must
-include a `p4runtime_translation` annotation so the compiler can mark
-occurrences of these values appropriately, so the agent can know which
-values need this translation.  See the example P4 header definition
-below.
-
-```
-[INCLUDE=examples/psa-example-parser-error-handling.p4:PortId_Annotation_Example]
-```
+The file `psa.p4` contains an annotation `p4runtime_translation` on
+the `type` definition of types `PortId_t` and `PortIdInHeader_t`.
+This enables the compiler to mark all uses of values of these types
+that are accessible from the P4Runtime API, so the agent software
+knows to translate them.  The benefit is that you do not need to put
+any special markings on your uses of values of these types throughout
+your P4 program.
+
+The cost of this approach is: if you want to do arithmetic on values
+of these types, you must explicitly cast them to a normal `bit<W>`
```

Comment:
normal is not defined. builtin type? core type?

Reply:
Next commit will remove "normal", leaving it simply "must explicitly cast them to a `bit<W>` type"